modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -94,4 +175,9 @@ protected boolean isValidExtension(String fileName) {
 		return false;
 	}
 
+	private static final Pattern _commentsPattern = Pattern.compile(
+		"(<%--([0-9]*[\\*\\w\\s\\-\\:\\(\\)\\.\\,\\/]+)--%>)");
```

Comment:
@TamyrisTorres, can we simplify it?

For example: `(\/\*)+(\n|.)*(\*\/)+`

Reply:
resolved